src/test/java/cmput402/tdd/AppTest.java

```diff
@@ -227,4 +228,257 @@ public void testDisplayPeople() {
         
     }
 
+    @Test
+    public void testEditRecipe(){
+        App app = new App();
+        Recipe recipe = new Recipe("apple pie");
```

Comment:
Does Recipe and Item need to be instantiated here? Can we use mocking instead?

Reply:
Mocked items, recipe remains instantiated.

Reply:
I don't see where you are mocking Items?